lib/plausible/auth/totp.ex

```diff
@@ -82,6 +83,11 @@ defmodule Plausible.Auth.TOTP do
     user.totp_enabled and not is_nil(user.totp_secret)
   end
 
+  @spec initiated?(Auth.User.t()) :: boolean()
+  def initiated?(user) do
+    not user.totp_enabled and not is_nil(user.totp_secret)
```

Comment:
```suggestion
    not user.totp_enabled and is_binary(user.totp_secret)
```

reads a bit better in this case